Lib/ufo2ft/__init__.py

```diff
@@ -153,8 +168,19 @@ def compileTTF(
 
     *layerName* specifies which layer should be compiled. When compiling something
     other than the default layer, feature compilation is skipped.
+
+    *skipExportGlyphs* is a list or set of glyph names to not be exported to the
+    final font. If these glyphs are used as components in any other glyph, those
+    components get decomposed. If the parameter is not passed in, the UFO's
+    "public.skipExportGlyphs" lib key will be consulted. If it doesn't exist,
+    all glyphs are exported. UFO groups and kerning will be pruned of skipped
+    glyphs **in-place**.
```

Comment:
@madig I'm not sure what you mean in this context by "in-place"... It's also confusing because there's another parameter with the same name. What's the relationship between the two, if any?

Reply:
Oh I see now. You modify the input UFO groups/kerning instead of creating a new copy and passing it on to the feature writers. Ideally the inplace argument would have an effect on the modification of groups/kerning. But i'm ok if we implement this later.